bigquery/tests/unit/test_external_config.py

```diff
@@ -56,15 +55,29 @@ def test_api_repr_base(self):
                 ],
             },
         })
-        ec = ExternalConfig.from_api_repr(resource)
+        ec = external_config.ExternalConfig.from_api_repr(resource)
         self._verify_base(ec)
-        self.assertEqual(ec.schema,
-                         [SchemaField('full_name', 'STRING', mode='REQUIRED')])
+        exp_schema = [
+            schema.SchemaField('full_name', 'STRING', mode='REQUIRED')
+        ]
+        self.assertEqual(ec.schema, exp_schema)
         self.assertIsNone(ec.options)
 
         got_resource = ec.to_api_repr()
         self.assertEqual(got_resource, resource)
 
+    def test_to_api_repr_base(self):
+        ec = external_config.ExternalConfig('')
+        ec.source_uris = self.SOURCE_URIS
+        ec.max_bad_records = 17
+        ec.autodetect = True
+        ec.ignore_unknown_values = False
+        ec.compression = 'compression'
+
+        got_resource = ec.to_api_repr()
+
+        self.assertEqual(got_resource, self.BASE_RESOURCE)
```

Comment:
Don't use `BASE_RESOURCE` here, use a dictionary with keys and values instead.

Even if it's redundant with a shared value, a test should contain everything you need to understand it's correctness. Your `test_to_api_repr_sheets` is an example of a test that does this right.